GOALS

```diff
@@ -1,5 +1,5 @@
 ============================================
-Chapel Public Release Goals (version 1.10.0)
+Chapel Public Release Goals (version 1.11.0)
 ============================================
 
 TODO: Update these goals.
```

Comment:
Yeesh, was this in there in the last release?  Let's remove it.  These goals are good enough for now.